src/mtx/commands/downlink/setOpParams.ts

```diff
@@ -0,0 +1,374 @@
+/**
+ * Downlink command to set device operator parameters.
+ *
+ * @example
+ * ```js
+ * import * as setOpParams from 'jooby-codec/mtx/commands/downlink/setOpParams.js'; * import {CommandBinaryBuffer} from 'jooby-codec/mtx/index.js';
+ *
+ * const parameters = {
```

Comment:
there was getDefaultOpParams or something like this in buffer

maybe we can remove all this copy-paste?

Reply:
copy-paste removed here and in uplink/getOpParams.ts